code/game/objects/items/implants/implantchair.dm

```diff
@@ -190,6 +190,9 @@
 /obj/machinery/implantchair/brainwash/implant_action(mob/living/C, mob/user)
 	if(!istype(C) || !C.mind) // I don't know how this makes any sense for silicons but laws trump objectives anyway.
 		return FALSE
+	if(is_banned_from(C.ckey, ROLE_BRAINWASHED))
+		to_chat(user, span_alert("[C] doesn't seem to respond to the brainwashing!"))
+		return FALSE
```

Comment:
Being banned should not confer metaprotection from being brainwashed. This could lead to people requesting a brainwash ban specifically to become immune to brainwashing.

It should instead kick them out of their body and offer them up to ghosts similar to how other conversions are handled.